lib/service/map_constant.dart

```diff
@@ -22,5 +22,6 @@ enum DrivingMode {
   driving,
   walking,
   bicycling,
-  transit
+  transit,
+  shuttle
 } // Do not use toString method here to get value
```

Comment:
delete this comment as it is unused

Reply:
Necessary because if the user uses DrivingMode.transit.toString(), it causes unexpected behaviours